src/test/emulators/auth/misc.spec.ts

```diff
@@ -365,6 +365,9 @@ describeAuthEmulator("accounts:query", ({ authApi }) => {
 });
 
 describeAuthEmulator("emulator utility APIs", ({ authApi }) => {
+  // TODO(lisajian): Add test that errors when changing allowDuplicateEmails config on tenant project
```

Comment:
Is this test possible at all? How would one set `tenantId` in the request if it is neither in the path or the request body?